ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -77,7 +77,17 @@ yum install -y git
 mkdir -p /tmp/artifacts
 
 if [ ! -e dev-scripts ] ; then
-  git clone https://github.com/openshift-metal3/dev-scripts.git
+  # If OCP CI is testing a dev-scripts PR, we check it out here.
+  # FIXME: Maybe move to built images containing dev-scripts contents?
+  if [[ "$REPO_NAME" == "dev-scripts" && "$PULL_NUMBER" != "" ]]; then
+    git clone https://github.com/openshift-metal3/dev-scripts.git
+    pushd dev-scripts
+    git fetch origin pull/$PULL_NUMBER/head:pr
+    git merge pr
+    popd
```

Comment:
If you put the contents of the dev-scripts image *in* the container image, then all of this isn't neccessary, you can scp dev-scripts from the container over to packet

Reply:
I just fetched the changes from your PR, if you see this step is excluded from the current workflow. I'm already putting the src in the build root image that will server to generate the `devscript` images, so this stuff will be removed.